ICRCs/ICRC-97/ICRC-97.md

```diff
@@ -0,0 +1,101 @@
+| ICRC |                   Title                    |                                      Author                                      |                Discussions                | Status |      Type       | Category |  Created   |
+|:----:|:------------------------------------------:|:--------------------------------------------------------------------------------:|:-----------------------------------------:|:------:|:---------------:|:--------:|:----------:|
+|  97  | Non-Fungible Token (NFT) Metadata Standard | Thomas (@sea-snake), Austin Fatheree (@skilesare), Dieter Sommer (@dietersommer) | https://github.com/dfinity/ICRC/issues/97 | Draft  | Standards Track |          | 2024-08-13 |
+
+# ICRC-97: Non-Fungible Token (NFT) Metadata Standard
+
+ICRC-97 is a metadata standard for the implementation of Non-Fungible Tokens (NFTs) on
+the [Internet Computer](https://internetcomputer.org).
+
+The standard is designed to allow for multiple assets with different data types with various purposes. Where additional
+asset purposes and attribute display types can be standardized in ICRC-97 extensions.
+
+A list of asset purposes and attribute display types are defined in ICRC-97 to the minimum needed for wallets and marketplaces to display NFTs.
+
+## Extending the ICRC-7 Standard with Token Metadata
+
+The [`icrc7_token_metadata`](https://github.com/dfinity/ICRC/blob/main/ICRCs/ICRC-7/ICRC-7.md#icrc7_token_metadata)
+method returns the token metadata for `token_ids`, a list of token ids. Each tuple in
+the response vector comprises an optional `metadata` element with the metadata expressed as a vector of `text` and `Value`
+pairs. Where `Value` is recursive as defined in the ICRC-3 standard.
+
+ICRC-7 does not specify the representation of token metadata any further than that it is represented in a generic manner
+as a vector of (text, Value)-pairs. The ICRC-97 standard defines a token metadata
+standard for ICRC-7 and other NFT standards.
+
+### Entrypoint
+
+The following metadata property MUST be defined in the root of the token metadata if we want to return on-chain
+metadata.
+
+| Root Metadata Property | ICRC-3 Type     | Description                            |
+|------------------------|-----------------|----------------------------------------|
+| icrc97:metadata        | variant { Map } | Contains on-chain metadata properties. |
+
+Alternatively the following metadata property MUST be defined in the root of the token metadata if we want to return
+external
+off-chain metadata instead of on-chain metadata.
+
+| Root Metadata Property   | ICRC-3 Type     | Description                                       |
+|--------------------------|-----------------|---------------------------------------------------|
+| icrc97:external_metadata | variant { Map } | Contains below external JSON metadata properties. |
```

Comment:
this does not prevent defining both, right?

Reply:
The metadata ICRC-3 structure doesn't prevent any dev from not following the spec, but you're right, according to the ICRC-97 spec only of these should be defined. I'll clarify this in the spec.